cves/kernel/CVE-2016-8630.yml

```
@@ -448,7 +497,14 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
+    The commit that caused this issue was a “fix” to the code that was originally
+    there. The commit message ("KVM: emulate: do not initialize memopp") removes
+    a check for non-NULL under incorrect assumptions. An undefined instruction 
+    with a ModR/M byte with Mod=0 and R/M-5 (e.g. 0xc7 0x15) will attempt to 
+    dereference a null pointer here. This seems to have been a mistake in 
+    understanding how the code worked or what assumptions were incorrect.
+
```

```suggestion
  answer: |
    The commit that caused this issue was a "fix" to the code that was originally
    there. The commit message ("KVM: emulate: do not initialize memopp") removes
    a check for non-NULL under incorrect assumptions. An undefined instruction 
    with a ModR/M byte with Mod=0 and R/M-5 (e.g. 0xc7 0x15) will attempt to 
    dereference a null pointer here. This seems to have been a mistake in 
    understanding how the code worked or what assumptions were incorrect.
```

Good, but I still think you could take some more time to expand on this. Wording could be improved.